src/importexport/midiimport/internal/midiimport/importmidi_simplify.cpp

```diff
@@ -41,7 +41,7 @@ bool areDurationsEqual(
         sum += ReducedFraction(d.second.fraction()) / d.first;
     }
 
-    return desiredLen == desiredLen;
+    return sum == desiredLen;
```

Comment:
See #7180, which does the exact same thing

Reply:
but needs a rebase, so I'll close mine.

Reply:
See also #7179 for 3.x, and the problems that went along with it, 2 mtests crashing

Reply:
See also https://musescore.org/en/node/314899, where this came up before

Reply:
Try whether ther MIDI mtests / imports "voice_tuplet.mid" and "voice_3_5_7_tuplets.mid" work with this change (they crash with the corresponding change in 3.x, see #7179)